### DISCUSSION TITLE
Final projects - research directions

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#24] [Announcements]
Post __three__ themes that you would like to explore for your [Final Project](https://github.com/periode/augmenting-gallery/wiki/Assignments#final-project---40), describing the theme, describing why you think it would be interesting to your audience. It can be either of:

- A __knowledge field__: how do people do things? What instruments, knowledge, practice, physics, cultures and contexts are involved in a particular know-how?
- A collection: what group of things (ranging from objects, images, songs, stories, places, etc.) can be linked together under a broader theme? What kind of collection would represent the themes of, e.g., "love", "nature", "conflict", "language"?

**Comment by chaerin-d** [comment]
One theme that that I would like to explore is "food", and I want to create an app of the restaurant made up by me and pretend I'm going to sell food in Berlin. This will be an interesting project to analyze market trends in Berlin and creativity. 

The second theme that I would like to create a collection of is "Classical Music in Berlin", especially about conductors of Berlin Philharmoniker. This interests me, who likes classical music and often attends concerts in Philharmonie, but Berliner Philharmoniker is one of the best orchestras in the world which can be attractive to others. It will be interesting to see the collections of the conductors.

The last theme that I would like to develop is  "life in Berlin", which is about collections of pictures that I took in Berlin and this can be displayed like a digital gallery through the app. 

* These are just my ideas for the final projects, and I would like to work with Ambra and Robbin. We will discuss more about what we want to work on for the project :)

**Comment by Caythian** [comment]
1. The first thing I want to do is to develop a website dedicated to showcasing the most breathtaking views in "Zelda: Tears of the Kingdom." Even half a year after its release, "Tears of the Kingdom" (ToTK) continues to captivate players with its innovative mechanics, expansive open-world concept (an evolution from "Breath of the Wild" with an even broader realm to explore), and compelling storyline. Beyond these elements, the game's stunning landscapes, ranging from sky islands to the mysterious underground, are equally mesmerizing. This website will highlight the game's most picturesque locations. Ideally, it will feature a game map, pinpointing the coordinates of these must-visit spots.
2. Also, I aim to develop a website that educates individuals on bird-watching, particularly in urban settings. Typically, bird-watching evokes images of venturing deep into forests. However, for those like me who might have limitations preventing them from exploring nature, urban bird-watching offers an alternative. This subject is rarely addressed, highlighting the need for such a platform. By teaching individuals to observe the avian life around them, the website will emphasize the interaction between humans and animals and shed light on the delicate urban ecosystem.
3. Lastly, I would like to make a collection of the prevalent trees found in Berlin. During the fall season, I discovered various chestnuts and nuts, which are seldom seen in New York or China. This led me to recognize Berlin's diverse tree species. I plan to design a website dedicated to these trees, featuring their images, essential facts, and interactive elements like a game to assist users in identifying which seed corresponds to which tree.

**Reply by Ambraaaliu** [reply]
!! I am really interested in your first idea!!! Really looking forward to see it (if you'll do this :)

**Comment by zbyiii** [comment]
1. I would like to explore the theme of nature. I am interested in plant and trees. Berlin is one of the greenest cities in Europe. And there are many kinds of trees in Berlin. I noticed that there are many old trees in Berlin, which have witnessed the history of the city. The oldest tree in Berlin has been there for hundreds of years. However, in recent years, the city is experiencing drought, and it is hard for trees to get enough water. There are organizations call on citizens to water the trees, and there is a interactive map on the website. I am interested in making a website to collect the normally seen species of plant and raise the awareness of preserving the green space. 
2. I would also like to work on the collection of wildlife in Berlin. There are different wild animals that lives in Berlin, such as beaver, fox, boar, falcon and so on. I think it would be interesting to make a collection of the wild animals live in Berlin, which shows a different aspect of the city aside of its rich culture and history.  
3. I am also interested in the traditional dance and operas in China. There are many types of traditional dance in China. They originated in different historical period and in different ethnic groups. Also, the opera culture in China is diverse and interesting. I would like to investigate on the origins and characteristics of different types of dance and opera, as well as the historical background of them.

**Comment by lizalmonds** [comment]
“Food”. It would be a collection of meals one can make at home. Sort of like an online cookbook for people to refer back to. Especially for those that are on a budget and places to go to buy affordable ingredients. 

“Music”. Influence of Jazz around the world and how to affected other genres. I would look into where Jazz is currently most popular and hwy. It's be a collection of popular contemporary artists in top 10 countries that listen to the most jazz.  

“Movement”. It would be a collection of footage I gathered during my time studying abroad. It would consist of stills of scenes from my travels and other videos I find online that depict movement utilizing modes of transportation and/or walking, specifically getting from point A to point B.

**Comment by bhornik** [comment]
1. Music: A collection of “scandalous” album+song covers by decade. This idea puts the notion that uncensored 21st century music is negatively impacting the youth when compared to previous decades into question. This collection could potentially prove or disprove the previous statement, but regardless, it will very clearly and visually represent a (more often than not) baseless claim about the current state of music, and more broadly track cultural trends over time.
2. Memes: A collection of “catastrophe memes.” The internet’s first response to catastrophe in both national and global news is usually expressed in meme format. This idea explores how humor is used to cope with catastrophe and the implications of coping using these questionable methods. The inherent challenge with this is how to present this intellectually instead of just making light of horrible events.
3. Warning Labels: A collection of cancer warnings on as broad a range of objects as possible. This idea brings into question how overexposure of signage can lead to the desensitization of incredibly clear warnings. Does the quantity of warning labels cause the consumer to ignore them entirely? How do we visually represent something as more dangerous than others?

**Comment by alkashiwag** [comment]
1.  Twitter lore: A collection focused on viral users, exchanges, and tweets that have found their way to legendary status in twitter. Meant to showcase that there is sometimes no rhyme or reason to why certain things are memorable. Twitter has also become one of the main forms of interacting with social media and the character limitations help users display creativity and wittiness.
2.  Popular music and beef: A collection on the many disputes between high profile artists. These disputes create massive headlines as well as extremely high selling albums and singles. Can also show how drama creates artist buzz.
3. NYC streetwear: A collection combining a decade of the  most prominent parts of the NYC streetwear fashion scene that shaped popular culture around the world. All of these stories show how doing your own thing can help you to identify with something much greater.

**Comment by Ambraaaliu** [comment]
1. East Asian Traditional Music Instruments: it may connect with my Guqin idea, I want to explore more about these East instruments' story and write a storyline about them. Hope this will evoke people's interest and attention in traditional music, which will help us to protect our cultural heritage. 
2. Anime History and Story: I want to focus more on Japanese Anime, especially Animation works from the period after World War II to the millennium. I would like to explore the development process of modern Japanese animation as an entertainment industry that affects East Asia and even the world, discuss how it is influenced by the historical background and new technology in its works, and then influence the entire animation industry, current artistic creation and popular culture.
3. Pop Music and fashion. I want to make a website about the mutual influence of pop music and fashion, and combine the development of media, set up a parallel time line, analyze and compare, and show people the development and mutual influence of modern pop culture

**Comment by liunikolai** [comment]
New Wave cinema: Documentation of the genesis, characteristics, and filmography of New Wave movements in film worldwide. What preexisting filmmaking traditions were they rebelling against, and how?

Diasporic art: Many of the most prominent Chinese artists developed and base their careers abroad due to heavy censorship and political hostility in their native country. I want to make a collection of Chinese diasporic art and works from other artists who left their homeland to pursue greater artistic freedom. 

Unrecognised ethnic groups: There are officially 56 ethnic groups in China, but in fact, there are many more. I am interested in making an ethnographic collection of these groups through pictures. The project can expand to unrecognised ethnic minorities in other countries. 

Music: A collection of my favourite albums from different countries (at least 45) supplemented with related visuals.

**Comment by RobbinHezhengWu** [comment]
1. Collection of every Porsche 911 road car
2. Collection of Every Albums from the artist Kafe Hu
3. Research of 1980s Hong Kong Kowloon Walled City
